my_list.rb

```diff
@@ -0,0 +1,28 @@
+require_relative './my_enumerable'
+
+# Class MyList
+class MyList
+  include MyEnumeralbe
+  def initialize(list)
+    @list = list
+  end
+
+  def each
+    i = 0
+    while i < @list.length
+      yield @list[i]
+      i += 1
+    end
+    @list
+  end
+end
+
+list = MyList.new([1, 2, 3, 4])
+
+puts(list.all? { |e| e > 5 })
+puts(list.all? { |e| e > 5 })
+
+puts(list.any? { |e| e == 2 })
+puts(list.any? { |e| e == 5 })
+
+puts(list.list.filter(&:even?))
```

Comment:
- [ ] This test doesn't seems to work. But this will work: ```puts(list.filter {|e| e.even?})```

So I will suggest you edit the highlighted line to reflect this change.